test/e2e/lib/gutenberg/gutenberg-editor-sidebar-component.js

```diff
@@ -224,6 +229,7 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 	}
 
 	async scheduleFuturePost() {
+		await this.expandStatusAndVisibility();
```

Comment:
Curious about this. Could you elaborate on why it was needed?

Reply:
On Gutenberg edge, the panel appeared to be collapsed sometimes and the test would fail. I added this to ensure the panel would be expanded so the UI for scheduling a post could be found.